docs/tutorials/11_VarQTE.ipynb

```diff
@@ -0,0 +1,957 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "lTiEsmieVAf-"
+   },
+   "source": [
+    "# Variational Quantum Time Evolution\n",
+    "This notebook demonstrates how to use the Variational Quantum Time Evolution (VarQTE) algorithm for computing the time evolving state under a given Hamiltonian. Specifically, it introduces variational quantum imaginary and real time evolution based on McLachlan's variational principle, and shows how this can be leveraged using the `algorithms.time_evolvers.variational` sub-module."
```

Comment:
```suggestion
    "This notebook demonstrates how to use the Variational Quantum Time Evolution (VarQTE) algorithm for computing the time evolving state under a given Hamiltonian. Specifically, it introduces variational quantum imaginary and real time evolution based on McLachlan's variational principle, and shows how this can be leveraged using the `qiskit_algorithms.time_evolvers.variational` sub-module."
```
Or drop qiskit_algorithms altogether - I would hope the context was self-evident enough but if its thought not...